File: StykerDG.FarmForge.DataModel/Models/Payment.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace StrykerDG.FarmForge.DataModel.Models
{
    [Table("Payment")]
    public class Payment : BaseModel
    {
        [Key]
        public int PaymentId { get; set; }
        public int OrderId { get; set; }
        public double Amount { get; set; }
    }
}

File: StrykerDG.FarmForge.Client/farmforge_client/lib/models/sales/payment.dart
class Payment {
  int paymentId;
  int orderId;
  double amount;

  Payment({
    this.paymentId,
    this.orderId,
    this.amount
  });

  factory Payment.fromMap(Map<String, dynamic> data) {
    return Payment(
      paymentId: data['PaymentId'],
      orderId: data['OrderId'],
      amount: data['Amount']
    );
  }

  Map<String, dynamic> toMap() {
    return {
      'PaymentId': this.paymentId,
      'OrderId': this.orderId,
      'Amount': this.amount
    };
  }
}